src/adapters/simpleble-adapter.ts

```diff
@@ -53,6 +53,24 @@ export class SimplebleAdapter extends EventEmitter implements BluetoothAdapter {
     private descriptors = new Map<string, string[]>();
     private charEvents = new Map<string, (value: DataView) => void>();
 
+    getHardwareAdapters() {
```

Comment:
```suggestion
    public getHardwareAdapters() {
```
Please make this explicitly public